### PR TITLE
#163724975 Add location field to room model

### PR DESCRIPTION
**Pull request by bevkololi**
#### What does this PR do?
Add location field to the room model.

#### Description of Task to be completed?
For the new structure, rooms created should belong to a particular location. One should therefore be able to add a location during room creation.

#### How should this be manually tested?
* Clone the branch and follow the following steps to [setup](https://github.com/andela/mrm_api).
* git checkout to `ch-add-location-in-rooms-163724975` and run the following mutations:

Create room:
```
mutation {
        createRoom(
            name: "Test3", roomType: "Meeting", capacity: 4, floorId: 2, officeId: 1, roomTags: [1], locationId:1,
      calendarId:"andela.com_3836323338323230343935@resource.calendar.google.com",
            imageUrl: "https://www.officelovin.com/wp-content/uploads/2016/10/andela-office-main-1.jpg") {  # noqa: E501
            room {
                id
                name
              	locationId
                roomType
                capacity
                floorId
                imageUrl
                roomTags {
                  name
                  description
                }
            }
        }
    }
```

#### Any background context you want to provide?
This conforms to the new structure of using Tags, that is LocationCreation-->RoomCreation-->AdditionOfTags.

#### Screenshots
* Creating a room in a particular location:
screenshot 2019-02-05 at 16 58 42

* Creating a room with an invalid location id
screenshot 2019-02-05 at 16 59 08

#### What are the relevant pivotal tracker stories?
[#163724975](https://www.pivotaltracker.com/story/show/163724975)

### CODE REVIEW

**Comment by kampkelly** [comment]
Looks good to me.

File: tests/test_rooms/test_room_crud_with_tags.py [hunk 1]
```diff
@@ -66,3 +67,12 @@ def test_query_room_by_id_with_tags(self):
             self,
             room_query_by_id,
             room_query_by_id_response)
+
+    def test_create_room_with_invalid_locationId(self):
+        """
+        Testing for room creation with invakid location id
```
Comment (caveinn) [inline tests/test_rooms/test_room_crud_with_tags.py]: Nice work. However, you have a typo here.

File: tests/test_rooms/test_room_crud_with_tags.py [hunk 2]
```diff
@@ -11,7 +11,8 @@
     rooms_query,
     query_rooms_response,
     room_query_by_id,
-    room_query_by_id_response
+    room_query_by_id_response,
+    room_invalid_LocationId_mutation
```
Comment (caveinn) [inline tests/test_rooms/test_room_crud_with_tags.py]: Check on the naming. You have mixed snake case and camel case.

**Comment by dnuwa** [comment]
Good work @bevkololi though you'll have to look into `room_invalid_LocationId_mutation` naming in the fixtures.

File: tests/test_rooms/test_room_crud_with_tags.py (hunk 1 above)
Comment (bevkololi) [reply tests/test_rooms/test_room_crud_with_tags.py]: Thanks @caveinn, worked on this.

**Pull request merged**